api/agent/server.go

```diff
@@ -574,17 +574,26 @@ func (s *Server) GetRouter() http.Handler {
 	// SPIRE server healthcheck
 	apiRtr.HandleFunc("/api/debugserver", s.debugServer)
 	apiRtr.HandleFunc("/api/healthcheck", s.healthcheck)
+	apiRtr.HandleFunc("/api/v1/debugserver", s.debugServer)
+	apiRtr.HandleFunc("/api/v1/healthcheck", s.healthcheck)
 
 	// Agents
 	apiRtr.HandleFunc("/api/agent/list", s.agentList)
 	apiRtr.HandleFunc("/api/agent/ban", s.agentBan)
 	apiRtr.HandleFunc("/api/agent/delete", s.agentDelete)
 	apiRtr.HandleFunc("/api/agent/createjointoken", s.agentCreateJoinToken)
+	apiRtr.HandleFunc("/api/v1/agents/list", s.agentList)
+	apiRtr.HandleFunc("/api/v1/agents/ban", s.agentBan)
+	apiRtr.HandleFunc("/api/v1/agents/delete", s.agentDelete)
+	apiRtr.HandleFunc("/api/v1/agents/createjointoken", s.agentCreateJoinToken)
 
 	// Entries
 	apiRtr.HandleFunc("/api/entry/list", s.entryList)
 	apiRtr.HandleFunc("/api/entry/create", s.entryCreate)
 	apiRtr.HandleFunc("/api/entry/delete", s.entryDelete)
+	apiRtr.HandleFunc("/api/v1/entries/list", s.entryList)
+	apiRtr.HandleFunc("/api/v1/entries/create", s.entryCreate)
+	apiRtr.HandleFunc("/api/v1/entries/delete", s.entryDelete)
 
```

Comment:
would you be able to group these additional lines in one section below all the other apis? I would like all the new `v1` apis in the same place

Reply:
Also for each of these, let's also add `spire` into the path to more clearly distinguish SPIRE and Tornjak API calls. Can add these after the `v1` - so something like `/api/v1/spire/agents/list` and so on